lib/parallel_tests/cli.rb

```diff
@@ -211,6 +212,7 @@ def parse_options!(argv)
 
       files, remaining = extract_file_paths(argv)
       unless options[:execute]
+        files = extract_files_from_cucumber_profile(options[:cucumber_profile]) if (!files.any?)&&options.key?(:cucumber_profile)
```

Comment:
... also add here `raise "only supported for cucumber" unless @runner.name == 'cucumber'` instead of in extract_files_from_cucumber_profile